tests/helper/helper_backstage.go

```diff
@@ -1,5 +1,6 @@
 package helper
 
+//nolint:staticcheck // Dot import more readable in test files
```

Comment:
This disables all staticcheck rules, no? Please be more specific on the exclusion rules. Or as suggested earlier, I think a configuration would help better in terms of maintenance.